src/isar/config/default.ini

```diff
@@ -1,84 +1,134 @@
 [DEFAULT]
-robot_directory = isar_robot
+# Determines which robot package ISAR will attempt to import
+# Name must match with an installed python package in the local environment
+robot_package = isar_robot
 
-[network]
-allowed_origins = http://localhost:3001
+# Determines the local path in which results from missions are stored
+local_storage_path = ./results
 
-[environment]
-azure_tenant_id = 3aa4a235-b6e2-48d5-9195-7fcf05b459b0
-azure_client_id = b29bed99-7637-4fdc-8809-48c777a9b714
-jwt_decode_audience = api://b29bed99-7637-4fdc-8809-48c777a9b714
-azure_keys_url = https://login.microsoftonline.com/3aa4a235-b6e2-48d5-9195-7fcf05b459b0/discovery/v2.0/keys
-eqrobot_environment = local
-
-[fastapi]
-env = development
-run_host = localhost
-run_port = 3000
+# Timeout in seconds for direct HTTP requests made through the RequestHandler
+request_timeout = 10
+
+# Timeout in seconds for retrieving messages on a queue from state machine to API
+queue_timeout = 10
```

Comment:
Could this comment be written more clearly?

This timeout is actually only used while checking if there is a message on the queue. If there is no message on the queue after the timeout passes, the check will return False.

```suggestion
# Timeout in seconds for checking whether there is a message on a queue in the state machine
queue_timeout = 10
```